deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -2203,28 +2212,52 @@ def regular_file_cud(folder, log_monitor, file_list=['testfile0'], time_travel=F
         create_file(REGULAR, folder, name, content=content)
 
     check_time_travel(time_travel, monitor=log_monitor)
-    event_checker.fetch_and_check('added', min_timeout=min_timeout, triggers_event=triggers_event)
+
+    event_checker.fetch_and_check('added', event_mode=event_mode, min_timeout=min_timeout,
+                                  triggers_event=triggers_event)
```

Comment:
```suggestion
    event_checker.fetch_and_check('added', min_timeout=min_timeout,
                                  triggers_event=triggers_event, event_mode=event_mode)
```

Having changed `event_mode`of order, we must change it here too.

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1b7ea0a207fdaf1ba67ee56d13785ac575b1ae0a)